changelog.d/1-api-changes/add-pydio-feature

```diff
@@ -0,0 +1 @@
+Add a new team feature for Pydio support
```

Comment:
```suggestion
Add a new team feature for Pydio (Wire Cells) support
```
I'm not sure if we shouldn't state the product name to make this easier to understand for people outside of Wire. What do you think? :thinking: